From cc288a6e6b2cc2dccbd868a1c66a84c8e471390d Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 8 Sep 2014 13:07:15 -0400
Subject: [tools] perf: Remove min define from perf.h

Message-id: <1410181752-25631-191-git-send-email-jolsa@redhat.com>
Patchwork-id: 91826
O-Subject: [PATCH RHEL7.1 BZ1134356 190/307] perf tools: Remove min define from perf.h
Bugzilla: 1134356
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1134356
https://bugzilla.redhat.com/show_bug.cgi?id=1134356

upstream
========
commit 1b7ae1c941af21ec386ea7c7e812ba430652731c
Author: Jiri Olsa <jolsa@kernel.org>
Date: Mon May 5 12:34:12 2014 +0200

description
===========
It's defined in tools/perf/util/include/linux/kernel.h header.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/perf.h b/tools/perf/perf.h
index a547f38..50aa88d 100644
--- a/tools/perf/perf.h
+++ b/tools/perf/perf.h
@@ -203,12 +203,6 @@ static inline unsigned long long rdclock(void)
  */
 #define asmlinkage
 
-#define min(x, y) ({    \
- typeof(x) _min1 = (x);   \
- typeof(y) _min2 = (y);   \
- (void) (&_min1 == &_min2);  \
- _min1 < _min2 ? _min1 : _min2; })
-
 extern bool test_attr__enabled;
 void test_attr__init(void);
 void test_attr__open(struct perf_event_attr *attr, pid_t pid, int cpu,
-- 
1.7.1